clar2wasm/src/standard/standard.wat

```diff
@@ -3417,6 +3417,78 @@
         (i32.const 1) (local.get $output-offset) (i32.sub (local.get $writeptr) (local.get $output-offset))
     )
 
+    (func $compare_clarity_names (param $offset_a i32) (param $len_a i32) (param $offset_b i32) (param $len_b i32) (result i32)
+        ;; compare two clarity names and return -1 if a < b, 0 if a == b or 1 if a > b
+        (local $min_len i32) (local $a i32) (local $b i32)
+
+        (local.set $min_len (select (local.get $len_a) (local.get $len_b) (i32.lt_u (local.get $len_a) (local.get $len_b))))
+
+        (loop
+            (if
+                (i32.eq
+                    (local.tee $a (i32.load8_u (local.get $offset_a)))
+                    (local.tee $b (i32.load8_u (local.get $offset_b)))
+                )
+                (then
+                    (local.set $offset_a (i32.add (local.get $offset_a) (i32.const 1)))
+                    (local.set $offset_b (i32.add (local.get $offset_b) (i32.const 1)))
+                    (br_if 1 (local.tee $min_len (i32.sub (local.get $min_len) (i32.const 1))))
```

Comment:
What does this `1` do here? I thought we needed a label?

Reply:
I tend to not use labels when the branching is obvious. 

I changed it in a62c0f25588283dca0e1200935c6527fcb92037d

Reply:
Thanks, I think the label is more clear. I also didn't know that you could just use an integer here.